.github/workflows/ci.yml

```diff
@@ -24,28 +29,33 @@ jobs:
         run: ./ci/scripts/show-env.sh
       - name: Commit metadata
         run: ./ci/scripts/lint-commits.sh "$GITHUB_BASE_REF"
+        if: ${{ github.event_name == 'pull_request' }}
       - name: License headers
         run: ./ci/scripts/check-licence-headers.sh "$GITHUB_BASE_REF"
+        if: ${{ github.event_name == 'pull_request' }}
       - name: Executable bits
         run: ./ci/scripts/exec-check.sh
       - name: Non-ASCII characters
         run: ./ci/scripts/check-ascii.sh
       - name: Python (flake8)
         run: ./ci/scripts/python-lint.sh "$GITHUB_BASE_REF"
+        if: ${{ github.event_name == 'pull_request' }}
       - name: Python (mypy)
         run: ./ci/scripts/mypy.sh
       - name: Validate testplans with schema
         run: ./ci/scripts/validate_testplans.sh
       - name: C/C++ formatting
-        run: ./ci/scripts/clang-format.sh "$GITHUB_BASE_REF"
+        run: ./ci/bazelisk.sh test //quality:clang_format_check
       - name: Rust formatting
-        run: ./ci/scripts/rust-format.sh "$GITHUB_BASE_REF"
+        run: ./ci/bazelisk.sh test //quality:rustfmt_check
```

Comment:
I would maybe suggest to run `quality:rustfmt_fix` because contrary to `rustfmt_check`, it actually runs on the entire repository.

Reply:
I've updated a new `rustfmt_test` rule that runs `rustfmt` but with the `--check` argument. Please could you review again?